lib/FundDistribution/FundDistributionFields/FundDistributionFieldsContainer.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FieldArray,
+} from 'redux-form';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+import {
+  stripesConnect,
+} from '@folio/stripes/core';
+
+import {
+  RepeatableField,
+} from '@folio/stripes/components';
+
+import { getFundsForSelect } from '../../utils';
+import { baseManifest } from '../../manifests';
+import FundDistributionFields from './FundDistributionFields';
+
+const FundDistributionFieldsContainer = ({
+  resources, formValues, disabled, required, price,
+}) => {
+  const funds = getFundsForSelect(resources);
+
+  const renderSubForm = (elem, index, fields) => {
+    const fund = fields.get(index);
+    const fundCode = get(funds.find(({ value }) => value === fund.fundId), 'code');
+
+    return (
+      <FundDistributionFields
+        fundCode={fundCode}
+        funds={funds}
+        fund={fund}
+        price={price}
+        required={required}
+        elem={elem}
+        disabled={disabled}
+      />
+    );
+  };
+
+  return (
```

Comment:
it should be in presentation IMO

Reply:
I'm not sure. Because I maked `FundDistributionFields` like presentation.

Reply:
`renderSubForm` is part of container? I don't think so

Reply:
You are right. I moved it to `FundDistributionFields`